components/wifi/esp8266-driver/ESP8266Interface.h

```diff
@@ -47,6 +47,8 @@
  */
 class ESP8266Interface : public NetworkStack, public WiFiInterface {
 public:
+    using NetworkStack::get_ip_address;
+    using WiFiInterface::connect;
```

Comment:
Are we truly causing all external drivers to adapt to this new API?

We are not controlling all of those, so I would still consider whether this can be avoided.

Reply:
Removed